infra/src/main/resources/i18n/translations.fi.json

```diff
@@ -1804,5 +1804,31 @@
     },
     "dropdown": {
         "add-new": "Lisää uusi"
+    },
+    "privilege": {
+        "view-basic": "Geoviitteen käyttö",
```

Comment:
Muutin tämän nyt yksiselitteisemmälle "Geoviitteen käyttö" -nimelle, sillä ilman tätä Geoviitettä ei käytetä, pure and simple (eikä tämän oikeuden takana juuri mitään muuta ole.)